config/custom-environment-variables.json

```diff
@@ -6,7 +6,8 @@
     "level": "LOG_LEVEL"
   },
   "server": {
-    "port": "SERVER_PORT"
+    "port": "SERVER_PORT",
+    "secretToken": "SERVER_SECRET_TOKEN"
```

Comment:
I would nest it under "application" under the root config file, since server is reserved for HTTP server env variables

Reply:
we should have better name than secret token, so it will be easier to understand its purpose.

Reply:
done.
renamed `secretToken` to `authToken`.